src/Shared/ConsoleLogs/TimestampParser.cs

```diff
@@ -36,6 +34,7 @@ public static bool TryParseConsoleTimestamp(string text, [NotNullWhen(true)] out
                 }
             }
 
+            var timestamp = match.ValueSpan;
             result = new(content.ToString(), DateTimeOffset.Parse(timestamp.ToString(), CultureInfo.InvariantCulture));
```

Comment:
```suggestion
            result = new(content.ToString(), DateTimeOffset.Parse(timestamp, CultureInfo.InvariantCulture));
```

No reason to create an intermediate string here.